### PR TITLE
Tock 2.0: screen write update

### PR DESCRIPTION
**Pull request by alexandruradovici**
### Pull Request Overview

This pull request fixes a write issue for the screen driver.

### Testing Strategy

This pull request was tested using a CLUE nRF52840 board.

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make prepush`.

### CODE REVIEW

File: boards/clue_nrf52840/Makefile [hunk 1]
```diff
@@ -17,25 +17,12 @@ KERNEL_WITH_APP=$(TOCK_ROOT_DIRECTORY)/target/$(TARGET)/debug/$(PLATFORM)-app.el
 .PHONY: program program-apps
 
 program: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).bin
-	ifndef PORT
```
Comment (phil-levis) [inline boards/clue_nrf52840/Makefile:20]: Can you separate out this Makefile changes to a PR to master?

**Review by phil-levis — CHANGES_REQUESTED** [review]
Other than the Makefile changes, looks good to go.

File: boards/clue_nrf52840/Makefile (hunk 1 above)
Comment (alexandruradovici) [reply boards/clue_nrf52840/Makefile]: Sorry for that, I did not notice the Makefile was commited.

**Review by hudson-ayers — APPROVED** [review]
bors r+

**Pull request merged**